documentation/source/library_reference.rst

```diff
@@ -269,3 +269,14 @@ XGMII
     :members:
     :member-order: bysource
     :show-inheritance:
+
+Miscellaneous
+=============
+
+WaveDrom
+--------
+
+.. automodule:: cocotb.wavedrom
+    :members:
+    :member-order: bysource
+    :synopsis: A signal tracer for WaveDrom.
```

Comment:
How does this piece render?

Reply:
Like this: https://cocotb-cmarqu.readthedocs.io/en/add-wavedrom-to-lib-ref/library_reference.html#miscellaneous

Reply:
So, this line does not render at all?

Reply:
Ah, the synopsis line you mean... This is needed and apparently only used for the Module Index: https://cocotb-cmarqu.readthedocs.io/en/add-wavedrom-to-lib-ref/py-modindex.html